### PR TITLE
Fixed some information and typos on agload.md

### PR DESCRIPTION
**Pull request by markgomer**
Also changed nomination of "nodes" to "vertices" or "vertexes" in order to keep consistency and avoid confusion to the user.

### CODE REVIEW

**Comment by dehowef** [comment]
We don't need semicolons after every list member

**Comment by markgomer** [comment]
Hello @dehowef, I agree the semicolons are not important at all, but there are other parts of the PR that I consider important, like the default paths to the csv files when calling the function to load vertexes and edges, and the phrase by the end of the document "Create label `City` and load vertices from csv file. *** Note this CSV file has id field ***", which is wrong.

I can remove the aesthetic parts of the commit and leave just these ones I mentioned, if you would reconsider.

**Comment by dehowef** [comment]
@markgomer Valid, I kind of just went through and blazed through a bunch of PRs to remove the clutter. I also don't think that it's necessary to change the term node to vertice-- it just seems a bit ... redundant to do so to me? They already mean the same thing. I'll reopen this and you can go ahead and redo them. I might just go in and proofread all of the documentation myself at some point.

File: docs/intro/agload.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ load_labels_from_file('<graph name>',
                       false)
 ```
 
-Function `load_edges_from_file` can be used to load properties from the CSV file. Please see the file structure in the following. 
+Function `load_edges_from_file` can be used to load edges from the CSV file. Please see the file structure in the following.
```
Comment (markgomer) [inline docs/intro/agload.md:35]: Changed this to conform with the prior sentence, which said "(...) to load ***vertices*** from the CSV files. "

**Review by dehowef — APPROVED** [review]
Great! Thank you

**Pull request merged**